packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -0,0 +1,184 @@
+import {StyleType, View} from "@khanacademy/wonder-blocks-core";
+
+import {
+    boxShadow,
+    border,
+    semanticColor,
+    sizing,
+} from "@khanacademy/wonder-blocks-tokens";
+
+import {DismissButton} from "./dismiss-button";
+
+type AllowedStyleProps = {
+    /**
+     * The background color of the card, as a string identifier that matches a semanticColor token.
+     * This can be one of:
+     * - `"subtle"`, matching `semanticColor.core.background.base.subtle`: a light gray background, useful for cards that need to stand out from the page background.
+     * - `"default"`, matching `semanticColor.core.background.base.default`: a white background, useful for cards that are placed on a light gray page background.
+     *
+     * Default: `"default"`
+     */
+    backgroundColor?: "subtle" | "default";
+    /**
+     * The border radius of the card, as a string identifier that matches a border.radius token.
+     * This can be one of:
+     * - `"radius_080"`, matching `border.radius.radius_080`: a moderate border radius, useful for cards that need to have a slightly rounded appearance.
+     * - `"radius_120"`, matching `border.radius.radius_120`: a more pronounced border radius, useful for cards that need to have a more rounded appearance.
+     *
+     * Default: `"radius_080"`
+     */
+    borderRadius?: "radius_080" | "radius_120";
+    /**
+     * The padding inside the card, as a string identifier that matches a sizing token.
+     * This can be one of:
+     * - `"size_0"`, matching `sizing.size_0`: no padding, useful for cards that need to have content flush with the edges.
+     * - `"size_160"`, matching `sizing.size_160`: moderate padding, useful for cards that need to have some space between the content and the edges.
+     * - `"size_240"`, matching `sizing.size_240`: more padding, useful for cards that need to have more space between the content and the edges.
+     *
+     * Default: `"size_160"`
+     */
+    padding?: "size_0" | "size_160" | "size_240";
+};
+type Props = {
+    /**
+     * Optional styles to be applied to the root element and the dismiss button.
+     */
+    styles?: {
+        root?: StyleType;
+        dismissButton?: StyleType;
+    };
+    /**
+     * A ref that will be passed to the root element (i.e. the card container).
+     */
+    ref?: React.Ref<any>;
+    /**
+     * The HTML tag to use for the card container. By default, this is a `<div>`, but
+     * if the card is being used as a landmark region, you may want to set this to
+     * `<section>` and provide an appropriate `aria-label` via the `labels` prop.
+     */
+    tag?: keyof JSX.IntrinsicElements;
+    /**
+     * The content for the card.
+     */
+    children: React.ReactNode;
+    /**
+     * A set of localizable labels for this component, including a dismiss button
+     * and the card itself, if marked as an HTML region.
+     */
+    labels?: {
+        dismissButtonAriaLabel?: string;
+        cardAriaLabel?: string;
```

Comment:
When should an aria label for the card be provided? Is this something we could document guidelines for? (I haven't looked into what the best practices for cards are yet!)

Reply:
I added some typings and docs for this. Basically, it's when a Document structure role is used that allows `aria-label` (`section` or `figure` seemed the most appropriate for this).

Reply:
@marcysutton When should `section` or `figure` be used for cards? Might also be helpful to document what to consider when making that decision!